enhancements/hypershift/support-disabling-image-registry-via-capability.md

```diff
@@ -0,0 +1,523 @@
+---
+title: support-disabling-image-registry-via-capability
+authors:
+  - flavianmissi
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - deads2k
+  - jubittajohn
+  - sanchezl
+  - sjenning
+  - tjungblu
+  - wking # OTA/cvo expert, for awareness and potential feedback
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - deads2k
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - deads2k
+  - sjenning
+creation-date: 2024-12-19
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/XCMSTRAT-771
+see-also:
+  - "/enhancements/installer/component-selection.md"
+---
+
+# Disable the Image Registry via Capability
+
+## Summary
+
+In OCP Classic it's possible to disable various optional components at install
+time via [cluster capabilities](cluster-capabilities). This enhancement proposes
+introducing a capabilities API to Hypershift, with initial support for a single
+capability for disabling the `ImageRegistry`.
+
+[cluster-capabilities]: https://docs.openshift.com/container-platform/4.17/installing/overview/cluster-capabilities.html
```

Comment:
nit: in this enhancement context, you might want a local link, like `[cluster capabilities](/enhancements/installer/component-selection.md)`, because that will have more technical/design context than the admin-facing docs.  You have that already in `see-also`, but I'm not sure what additional context the version-pinned admin docs are adding.